bin/server.js

```diff
@@ -41,35 +42,49 @@ async function startServer () {
   app.use(cors({ origin: '*' }))
 
   // MIDDLEWARE END
-
   // Retrieve hash from BCH network and retrieve data from IPFS.
 
   // Get the latest hash off the BCH network.
   const hash = await bch.findHash()
+  // To-Do: Add some retry code here. If it doesn't find the hash on the first try,
```

Comment:
Can you look into adding the `p-retry` library to auto-retry if the hash isn't found on the first try?

Here is an example of how I used this library in a project recently, to retry 5 times before failing:
https://github.com/Permissionless-Software-Foundation/jwt-bch-api/blob/fbbac81ce6dc7844122ff44d2a85b4561369c7f7/src/lib/bch.js#L289-L302